### ISSUE TITLE
snapshot list is greyed out

### ISSUE THREAD

**Issue by oz123** [virt-manager/virt-manager#203]
**Distro**: gentoo

The snapshot list is greyed out, when I start the program from the CLI I see the following error:
```
$ virt-manager --no-fork
Exception ignored in: <function virDomainSnapshot.__del__ at 0x7fc1c944bcb0>
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/libvirt.py", line 7030, in __del__
    if self._o is not None:
AttributeError: 'virDomainSnapshot' object has no attribute '_o'
```
There are a few snapshots on an existing domain:
```
virsh # snapshot-list debian2 
 Name        Creation Time               State
--------------------------------------------------
 snapshot1   2020-12-10 22:05:15 +0100   running
 virgin      2020-12-10 22:04:20 +0100   running
```

Any idea how I can fix this?

**Comment by crobinso** [comment]
Thanks for the report, but you need to update libvirt-python, this bug was fixed in version 6.9.0

**Comment by oz123** [comment]
Thank you for the quick response. I submitted a patch to my distro's package...
